src/fides/api/ctl/database/database.py

```diff
@@ -35,6 +35,7 @@ def get_alembic_config(database_url: str) -> Config:
     return config
 
 
+# Prevent logging out sensitive values during failures
```

Comment:
Unfortunately I think I am still seeing the secrets, even when setting dev_mode to False. I think it might have something to do with Alembic raising the exception, maybe we can `try/except` here to either sanitize messaging or obfuscate with the loguru function you made

Reply:
oh interesting, thank you for figuring out how to reproduce!

I think the solution here then is to get `alembic` added to the logger then

Reply:
for reference later: https://stackoverflow.com/questions/61415220/change-alembic-logger